app/src/main/java/com/arygm/quickfix/ui/profile/becomeWorker/UpgradeToWorkerScreen.kt

```diff
@@ -102,7 +201,8 @@ fun BusinessScreen(
             modifier =
                 Modifier.padding(innerPadding).fillMaxSize().semantics {
                   testTag = C.Tag.upgradeToWorkerPager
-                }) { page ->
+                },
+            userScrollEnabled = true) { page ->
```

Comment:
I think this should be false and not true to ensure a good functionality

Reply:
Nice catch ! i forgot to put it back. Thanks !!